dispatch/httpdispatch.go

```diff
@@ -18,17 +18,26 @@
 package dispatch
 
 import (
+	"bytes"
+	"context"
+	"crypto/hmac"
+	"crypto/sha1"
+	"encoding/hex"
 	"fmt"
+	"io"
+	"io/ioutil"
 	"net/http"
 	"net/url"
+	"strconv"
 	"sync"
-	"sync/atomic"
 	"time"
 
-	"github.com/aws/aws-sdk-go/service/sqs"
 	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
 	"github.com/xmidt-org/mimisbrunnr/model"
+	"github.com/xmidt-org/webpa-common/logging"
 	"github.com/xmidt-org/webpa-common/semaphore"
+	"github.com/xmidt-org/webpa-common/xhttp"
 	"github.com/xmidt-org/wrp-go/v2"
 )
 
```

Comment:
I don't see anything in the dispatcher config that the dispatcher actually needs.  I think you can just remove it.